src/sql/src/plan.rs

```diff
@@ -105,7 +105,7 @@ pub enum Plan {
     DropObjects(DropObjectsPlan),
     EmptyQuery,
     ShowAllVariables,
-    ShowCreate(SendRowsPlan),
```

Comment:
Why did these changes need to be in this PR?

Reply:
We need to know the schema of the object we're looking at so that we can check that the user has `SCHEMA(U)` privileges. So I created a new plan for `ShowCreate`, `ShowCreatePlan`, that included the `GlobalId` of the object we're looking at. After that `SendRowsPlan` was not used anywhere so I deleted it.